src/main/java/com/wavefront/datastructures/StatSpan.java

```diff
@@ -0,0 +1,20 @@
+package com.wavefront.datastructures;
+
+import com.wavefront.sdk.common.Pair;
+import com.wavefront.sdk.entities.tracing.SpanLog;
+
+import java.util.List;
+import java.util.UUID;
+
+import javax.annotation.Nullable;
+
+public class StatSpan extends Span{
+  public StatSpan(String name, long startMillis, long durationMillis, @Nullable String source, UUID traceUUID, UUID spanUUID, @Nullable List<UUID> parents, @Nullable List<UUID> followsFrom, @Nullable List<Pair<String, String>> tags, @Nullable List<SpanLog> spanLogs, SpanKind kind) {
```

Comment:
I guess that you don't need spanKind parameter for this constructor and can just set `SpanKind.STATISTICS` for the base class.